modules/troubleshooting/pages/index.adoc

```diff
@@ -159,6 +147,71 @@ The output shows sites being linked and a service being exposed on a {service-ne
 However, this output is most useful when reporting an issue and is included in the {skupper-name} debug tar file.
 --
 
+.Additional information
+
+* xref:checking-links[]
+
+// Type: procedure
+[id='checking-links']
+== Checking links
+
+You must link sites before you can expose services on the {service-network}.
+This section outlines some advanced options for checking links.
+
+
+. Check the link status:
++
+--
+----
+$ skupper link status --namespace east
+
+Links created from this site:
+-------------------------------
+Link link1 is active
+----
+
+A link exists from the specified site to another site, meaning a token from another site was applied to the specified site.
+
+NOTE: Running `skupper link status` on a connected site produces output only if a token was used to create a link.   
+
+If you use this command on a site where you did not create the link, but there is an incoming link to the site:
+----
+$ skupper link status --namespace west
+
+Links created from this site:
+-------------------------------
+There are no links configured or active
+
+Currently active links from other sites:
+----------------------------------------
+A link from the namespace east on site east(536695a9-26dc-4448-b207-519f56e99b71) is active 
+----
+--
+
+. Check the verbose link status:
++
+--
+----
+$ skupper link status link1 --verbose --namespace east
+
+ Cost:          1
+ Created:       2022-10-24 12:50:33 +0100 IST
+ Name:          link1
+ Namespace:     east
+ Site:          east-536695a9-26dc-4448-b207-519f56e99b71
+ Status:        Active
+----
+
+The output shows detail about the link, including a timestamp of when the link was created and the associated relative cost of using the link.
+
+The status of the link must be `Active` to allow service traffic.
+--
+
+.Additional information
+
+* xref:checking-sites[]
+
+
```

Comment:
Should we reference the behaviour by default of `skupper link status link1` (non-verbose) as well?

Reply:
There's the simple case at  https://github.com/skupperproject/skupper-docs/pull/118/files#diff-390bcfec3d39389b5ae7b1dd9fe9edba79cbc980444d13c14a11e8ae96a620f5R166

For simple cases, I kinda implicitly reference the getting started, and so assume a single link from 'east'... ie didn't specifiy 'link1'. wdyt?

Reply:
You are right, I think it is good the way it is. Thanks!